### ISSUE TITLE
CM-December2022-Contribution Spanish Community-ERCAB24

### ISSUE THREAD

**Issue by ercab2408** [AstarNetwork/growth-program#2079]
![StatsTwitter Diciember-2](https://user-images.githubusercontent.com/76635320/210858941-271a86e8-d95e-4747-8049-5c693384be86.jpg)
![StatsTwitter Diciember-3](https://user-images.githubusercontent.com/76635320/210858963-2dc42219-e3a7-4966-b2fc-b2a06e6b446e.jpg)
Hi Community
My name es Ercab, as the community manager of a Astar&Shiden Network telegram and discord, my main job is to ensure that the protocol's user community is informed and engaged. This includes moderating the communication channels, responding to users' questions and issues, and organizing events as well as promoting the protocol through social media and other channels.

Last December, together with my ambassador friend @Van, we organized a Live with guests and we recounted the entire trajectory of Astar&Shiden in 2022 and the most relevant in announcements and alliances. links

We also participated as Astar Network in the Polkadot Latam day in Colombia where we contributed to the protocol connected to two project prospects Blumer and kastor Finance. See images and tweet

Data
Email:ercab24@protonmail.com
Twitter: @ercab24 @AstarNetwork_ES
Discord:Ercab24#2125
Telegram: @ercab24 / https://t.me/PlasmESP
Account Shiden rewards#: aJMU3fdQVv98WmcdCXpb6jcYeWkwjCGwQycefP8at1HQefu

In summary, as the community manager of a Astar&Shiden telegram and discord, my job is to ensure that the user community is informed and engaged and that the protocol continues to progress and improve through community feedback.
![FinalBanner](https://user-images.githubusercontent.com/76635320/210847636-086f4158-6e3a-4594-a780-4f3b612d74b6.png)
![Live Final year 2022](https://user-images.githubusercontent.com/76635320/210848243-81db0678-529f-41d0-86bc-6eab325463cb.jpg)
![PolkadotLatamworkshopday](https://user-images.githubusercontent.com/76635320/210848415-d1e34eca-9960-4f54-a857-5b37f7788b22.jpg)
![StatsTwitter Diciember](https://user-images.githubusercontent.com/76635320/210858903-554f737d-bdbc-4b6e-be5c-072146328019.jpg)
